ambry-clustermap/src/test/java/com.github.ambry.clustermap/MockHelixAdmin.java

```diff
@@ -164,6 +166,21 @@ void bringInstanceDown(String instance) {
   }
 
   /**
+   * Sets the state of a partition
+   * @param partition partition for which state needs to be updated
+   * @param partitionState {@link PartitionState} that needs to be set
+   */
+  void setPartitionState(String partition, PartitionState partitionState) {
+    for (IdealState entry : resourcesToIdealStates.values()) {
+      for (String partitionStr : entry.getPartitionSet()) {
```

Comment:
Calling this `partitionStr` is confusing since `partition` is also a partition string. Could you make this more readable?